posts/2021-11-09-now-with-more-elixir.md

```diff
@@ -0,0 +1,208 @@
+%{
+author: "Sean Callan",
+author_link: "https://github.com/doomspork",
+tags: ["Announcement"],
+date: ~D[2021-11-09],
```

Comment:
Not sure when you want to publish this, but it should probably be updated beforehand!